src/components/editor/editor-pane/editor-pane.tsx

```diff
@@ -54,12 +56,7 @@ export const EditorPane: React.FC<EditorPaneProps & ScrollProps> = ({ onContentC
   const { t } = useTranslation()
   const [editor, setEditor] = useState<Editor>()
   const [statusBarInfo, setStatusBarInfo] = useState<StatusBarInfo>(defaultState)
-  const [editorPreferences, setEditorPreferences] = useState<EditorConfiguration>({
-    theme: 'one-dark',
-    keyMap: 'sublime',
-    indentUnit: 4,
-    indentWithTabs: false
-  })
+  const editorPreferences = useSelector((state: ApplicationState) => state.editorConfig.preferences)
```

Comment:
```suggestion
  const editorPreferences = useSelector((state: ApplicationState) => state.editorConfig.preferences, equal)
```

and
```
import equal from `fast-deep-equal`
```

If you're retreiving an object with the useSelector, then please provide the equals function from fde.  Otherwise this selector would trigger every time the redux state gets updated, even if the preferences didn't change, because the js default equality check doesn't inspect the object's fields.
It's important to import equal from fast-deep-equal, because this is the only good one.

Reply:
Done in ddb3484